modules/setup/library/Setup/WebWizard.php

```diff
@@ -445,7 +445,7 @@ public function getSetup()
                         ? $pageData['setup_auth_db_creation']['password']
                         : null,
                     'schemaPath'        => Config::module('setup')
-                        ->get('schema', 'path', Icinga::app()->getBaseDir('etc' . DIRECTORY_SEPARATOR . 'schema'))
+                        ->get('schema', 'path', Icinga::app()->getBaseDir('schema'))
```

Comment:
This location does not exist. The schema files are still located in `etc/schema`.

Reply:
Done.